python/lsst/pipe/tasks/calibrate.py

```diff
@@ -21,151 +21,80 @@
 #
 import math
 
-import lsst.daf.base as dafBase
+from lsstDebug import getDebugFrame
 import lsst.pex.config as pexConfig
-import lsst.afw.math as afwMath
-import lsst.afw.table as afwTable
-import lsst.meas.algorithms as measAlg
 import lsst.pipe.base as pipeBase
-from lsstDebug import getDebugFrame
-from lsst.afw.display import getDisplay
-from lsst.meas.base import BasePlugin, SingleFrameMeasurementTask, MeasureApCorrTask
+import lsst.afw.table as afwTable
```

Comment:
This import of lsst.afw.table has been moved.  Please leave it where it was unless there's a good reason (and then do it on a separate commit).